benchmark/result/result.py

```diff
@@ -0,0 +1,36 @@
+"""Basic result classes for benchmarking."""
+
+from uuid import UUID
+
+from attrs import define, field
+from attrs.validators import instance_of
+from pandas import DataFrame
+
+from baybe.serialization.core import converter
+from baybe.serialization.mixin import SerialMixin
+from benchmark.definition.config import BenchmarkScenarioSettings
+from benchmark.result.metadata_class import ResultMetadata
+
+
+@define(frozen=True)
+class Result(SerialMixin):
+    """A single result of the benchmarking."""
+
+    benchmark_origin_identifier: UUID = field(validator=instance_of(UUID))
+    """The unique identifier of the benchmark running which can be set
+    to compare different executions of the same benchmark setting."""
+
+    benchmark_settings: BenchmarkScenarioSettings = field(
+        validator=instance_of(BenchmarkScenarioSettings)
+    )
+    """Settings about the benchmarking result."""
+
+    benchmark_result: DataFrame = field(validator=instance_of(DataFrame))
```

Comment:
this name is also somewhat redundant, in particular the "benchmark" part. Can we find an alternative?

Reply:
Maybe just `results_df`?

Reply:
both `result` and `benchmark` are redundant if we consider this is part of a class called `Result` that is applied to a  `Benchmark`
so what about simply `df`, `data`, `frame`, `content`, `raw`, `raw_df` etc